x-pack/plugins/infra/server/lib/adapters/framework/kibana_framework_adapter.ts

```diff
@@ -27,6 +26,7 @@ import { RequestHandler } from '../../../../../../../src/core/server';
 import { InfraConfig } from '../../../plugin';
 import type { InfraPluginRequestHandlerContext } from '../../../types';
 import { IndexPatternsFetcher, UI_SETTINGS } from '../../../../../../../src/plugins/data/server';
+import { TimeseriesVisData } from '../../../../../../../src/plugins/vis_type_timeseries/common/types';
```

Comment:
please fix import. We should import that type from '../../../../../../../src/plugins/vis_type_timeseries/server'